lib/galaxy/tool_util/deps/mulled/invfile.lua

```diff
@@ -101,11 +101,11 @@ inv.task('build')
 if VAR.SINGULARITY ~= '' then
     inv.task('singularity')
         .using(singularity_image)
-        .withHostConfig({binds = {"build:/data", "'" .. singularity_image_dir .. "':/import"}, privileged = true})
+        .withHostConfig({binds = {"build:/data", singularity_image_dir .. ":/import"}, privileged = true})
```

Comment:
Is this change also needed? I think the quotes were to support the case where `singularity_image_dir` contains spaces.

Reply:
Yes, we end up with `'singularity_image_dir'` on the file system otherwise. I've changed the test to include a space in `singularity_image_dir`.